content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -410,4 +410,8 @@ Defragmentation is an expensive operation, so it should be executed as infrequen
 as possible. On the other hand, it's also necessary to make sure any etcd member
 will not run out of the storage quota. The Kubernetes project recommends that when
 you perform defragmentation, you use a tool such as [etcd-defrag](https://github.com/ahrtr/etcd-defrag).
+
+It can also be useful to run the defragmentation tool as a Kubernetes CronJob, 
+[as documented here](https://github.com/ahrtr/etcd-defrag/blob/main/doc/etcd-defrag-cronjob.yaml)
+, to make sure that defragmentation happens regularly.
```

Comment:
This doesn't match our style guide, and it makes the note longer.

How about editing the README for https://github.com/ahrtr/etcd-defrag to mention the example manifest?

Reply:
I tried to improve the style according to https://kubernetes.io/docs/contribute/style/style-guide/

It's already mentioned in the README of the `etcd-defrag` tool. I however feel like this will make more people aware of the possibility to run a defragmentation tool in a CronJob, which is beneficial to cluster operators.

regarding the note length, it will indeed add one sentence, but that's the purpose of documentation to make information available where it can be useful, and that doesn't come without a new sentence 😅